api/fixtures/example.go

```diff
@@ -226,23 +196,100 @@ aws_secret_access_key = %s
-					Region: o.AWS.Region,
-					Roles:  o.AWS.Roles,
-					CloudProviderConfig: &hyperv1.AWSCloudProviderConfig{
-						VPC: o.AWS.VPCID,
-						Subnet: &hyperv1.AWSResourceReference{
-							ID: &o.AWS.SubnetID,
-						},
-						Zone: o.AWS.Zone,
+			Platform: hyperv1.PlatformSpec{},
+		},
+	}
+
+	switch o.PlatformType {
+	case "aws":
+		cluster.Spec.Platform.Type = hyperv1.AWSPlatform
+		cluster.Spec.SigningKey = corev1.LocalObjectReference{Name: signingKeySecret.Name}
+		cluster.Spec.Platform.AWS = &hyperv1.AWSPlatformSpec{
+			Region: o.AWS.Region,
+			Roles:  o.AWS.Roles,
+			CloudProviderConfig: &hyperv1.AWSCloudProviderConfig{
+				VPC: o.AWS.VPCID,
+				Subnet: &hyperv1.AWSResourceReference{
+					ID: &o.AWS.SubnetID,
+				},
+				Zone: o.AWS.Zone,
+			},
+			KubeCloudControllerCreds: corev1.LocalObjectReference{Name: kubeCloudControllerCredsSecret.Name},
+			NodePoolManagementCreds:  corev1.LocalObjectReference{Name: nodePoolManagementCredsSecret.Name},
+		}
+		cluster.Spec.Services = []hyperv1.ServicePublishingStrategyMapping{
+			{
+				Service: hyperv1.APIServer,
+				ServicePublishingStrategy: hyperv1.ServicePublishingStrategy{
+					Type: hyperv1.LoadBalancer,
+				},
+			},
+			{
+				Service: hyperv1.OAuthServer,
+				ServicePublishingStrategy: hyperv1.ServicePublishingStrategy{
+					Type: hyperv1.Route,
+				},
+			},
+			{
+				Service: hyperv1.OIDC,
+				ServicePublishingStrategy: hyperv1.ServicePublishingStrategy{
+					Type: hyperv1.Route,
+				},
+			},
+			{
+				Service: hyperv1.Konnectivity,
+				ServicePublishingStrategy: hyperv1.ServicePublishingStrategy{
+					Type: hyperv1.LoadBalancer,
+				},
+			},
+			{
+				Service: hyperv1.Ignition,
+				ServicePublishingStrategy: hyperv1.ServicePublishingStrategy{
+					Type: hyperv1.Route,
+				},
+			},
+		}
+
+	case "none":
+		cluster.Spec.Platform.Type = hyperv1.NonePlatform
+		cluster.Spec.Services = []hyperv1.ServicePublishingStrategyMapping{
+			{
+				Service: hyperv1.APIServer,
+				ServicePublishingStrategy: hyperv1.ServicePublishingStrategy{
+					Type: hyperv1.NodePort,
+					NodePort: &hyperv1.NodePortPublishingStrategy{
+						Address: o.Annotations["master_0.Address"],
 					},
-					KubeCloudControllerCreds: corev1.LocalObjectReference{Name: kubeCloudControllerCredsSecret.Name},
-					NodePoolManagementCreds:  corev1.LocalObjectReference{Name: nodePoolManagementCredsSecret.Name},
 				},
 			},
-		},
+			{
+				Service: hyperv1.OAuthServer,
+				ServicePublishingStrategy: hyperv1.ServicePublishingStrategy{
+					Type: hyperv1.Route,
+				},
+			},
+			{
+				Service: hyperv1.OIDC,
+				ServicePublishingStrategy: hyperv1.ServicePublishingStrategy{
+					Type: hyperv1.Route,
+				},
+			},
+			{
+				Service: hyperv1.Konnectivity,
+				ServicePublishingStrategy: hyperv1.ServicePublishingStrategy{
+					Type: hyperv1.NodePort,
+					NodePort: &hyperv1.NodePortPublishingStrategy{
+						Address: o.Annotations["master_0.Address"],
```

Comment:
After https://github.com/openshift/hypershift/pull/458, this can be `Route`

Reply:
ok makes sense